packages/themes/src/themes/shadcn.ts

```diff
@@ -1,6 +1,7 @@
 import { experimental_createTheme } from '../createTheme';
 
 export const shadcn = experimental_createTheme({
+  cssLayerName: 'components',
```

Comment:
Since we know shadcn is using TW, we can specify the cssLayerName to be defined within components, relying on TW layer order specified.